### PR TITLE
07 - Timer name and renaming Resolve #15

### LINKED ISSUE #15: 07 - Timer name and renaming

### PR DESCRIPTION
**Pull request by jaeyap**

### CODE REVIEW

File: components/Clock.tsx [hunk 1]
```diff
@@ -1,147 +1,147 @@
...
-    position: "absolute",
-    height: 4,
-    borderRadius: 4
-  },
-  lineBackground: {
-    position: "absolute",
-    backgroundColor: "none"
-  }
-});
+import { StyleSheet } from "react-native";
```
Comment (cloudsoh) [inline components/Clock.tsx:1]: What did you change in this file?

**Review by cloudsoh — CHANGES_REQUESTED** [review]

File: components/Clock.tsx (hunk 1 above)
Comment (jaeyap) [reply components/Clock.tsx:1]: I have no idea, I don't think I have changed anything in this file.

File: components/TimerNameInput.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import { TextInput } from "react-native";
+import { StyleSheet } from "react-native";
+
+export interface CountdownNumberInputProps {
```
[the remark this answers is not in the document]
Comment (jaeyap) [reply components/TimerNameInput.tsx:4]: Ok noted

File: components/Clock.tsx (hunk 1 above)
Comment (cloudsoh) [reply components/Clock.tsx:1]: If you think you didn't change and shouldn't have committed it, then can revert this file. Then next time check every file first before commit it.

I think this commit might be caused by csrf thingy or something. Can try running `git diff origin/master` in this branch to inspect the difference

**Review by cloudsoh — APPROVED** [review]

**Pull request merged**